checkov/terraform/checks/resource/gcp/CloudArmorWAFACLCVE202144228.py

```diff
@@ -38,6 +39,32 @@ def scan_resource_conf(self, conf: Dict[str, List[Any]]) -> CheckResult:
                         if rule.get("preview") == [True]:
                             return CheckResult.FAILED
                         return CheckResult.PASSED
+
+        resource_name = conf.get("name")[0]
```

Comment:
Will we always have a name? I know the name attribute can be omitted from the resource but I am not sure if populate this field in a default way in this case. I also saw in other locations we check if conf["name"] is null.

Reply:
Name is a required field for this resource: https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/compute_security_policy#name-1